presign/signup/templates/signup/root.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% load i18n %}
+
+{% block content %}
+    <div>
+        <h3>
+            There is currently no public event to sign up for.
+            {# TODO: Dynamically load the enabled events in case we want this #}
+        </h3>
+    </div>
+ 
```

Comment:
We should not duplicate the event list here (this should stay on jugendhackt.org/events). I'd prefer something like the pretix-style on `/`: "This is a presign installation. The control interface is [here](/control/)" and not have something on `/signup` (or redirect to `/`)

Reply:
Done